app/models/hub_edos/person_api/v1/affiliation.rb

```diff
@@ -0,0 +1,51 @@
+module HubEdos
+  module PersonApi
+    module V1
+      class Affiliation
+        # a short descriptor representing the kind of affiation, such as student or employee, etc.
+        attr_accessor :type
+
+        # a more detailed description for state of the affiliation, such as admitted or retired	string
+        attr_accessor :detail
+
+        # a short descriptor representing the state of the affiliation, such as active or inactive, etc.
+        attr_accessor :status
+
+        # the date this affiliation became effective
+        attr_accessor_with_lower_camelized_alias :from_date
+
+        MATRICULATED_TYPE_CODE = 'APPLICANT'
+        MATRICULATED_DETAILS_TO_EXCLUDE = [
```

Comment:
We already encountered some confusion with this name. Maybe `EXCLUDED_FROM_MATRICULATION` would be better. I think the use of `DETAILS` is just because of the implementation. Also I would go to the next step and define constants for the individual states.

```
SIR_COMPLETED_DETAIL='SIR Completed'
DEPOSIT_PENDING_DETAIL='Deposit Pending'

EXCLUDED_FROM_MATRICULATION=[SIR_COMPLETED_DETAIL, DEPOSIT_PENDING_DETAIL]

def matriculated_but_excluded?
  type.code == MATRICULATED_TYPE_CODE && EXCLUDED_FROM_MATRICULATION.include?(detail)
end
```

Reply:
I suspect this labeling is about matriculated status that is excluded from a specific application / context. I asked the BSA about it and they didn't know what it meant. Let's leave it as it is for now until someone knows and clarifies.